### DISCUSSION TITLE
2FA Without Phone?

### DISCUSSION THREAD

**Discussion by imdrogon** [community/community#73418] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I don't have any phone, but I'm required to enable 2FA apparently. Due to this dumb decision by GitHub, I might need to switch to a different platform.

**Comment by melvincwng** [comment, marked as answer]
@imdrogon, there are other ways to set up 2FA without using a phone. 

For more info, you can check out this [GitHub Documentation link](https://docs.github.com/en/enterprise-server@3.10/authentication/securing-your-account-with-two-factor-authentication-2fa/configuring-two-factor-authentication) over here.

**Comment by mukesh23950** [comment]
Two-factor authentication (2FA) enhances security by requiring two types of information from the user before they can access their account: something they know (like a password) and something they have (like a physical device). If you prefer not to use a phone as part of your 2FA method, there are several alternatives:

Authenticator Apps:
These apps generate time-based, one-time passcodes (TOTP) that users must enter in addition to their password. Popular authenticator apps include:

Google Authenticator
Authy
Microsoft Authenticator
LastPass Authenticator
Hardware Tokens:
These are physical devices that generate a passcode at the push of a button or when they are plugged into a computer. Examples include:

YubiKey
RSA SecurID
Thetis Fido U2F Security Key
Software Tokens:
Software tokens are applications that can run on various devices, including desktops or laptops, to generate a code for 2FA. They're similar to authenticator apps but don't require a mobile device.

Biometric Verification:
Some systems allow the use of fingerprints, facial recognition, or iris scans as a form of second-factor authentication.

Email-Based 2FA:
Some systems will send a code via email, which you must enter to log in. However, this method can be less secure if your email account is compromised.[ Read More Topic ](https://guideublog.com/)
